src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -286,9 +294,20 @@
       ApplicationDependencies="@(_BlazorManagedRuntimeAssembly)"
       WebAssemblyBCLAssemblies="@(_WebAssemblyBCLAssembly)">
 
-      <Output TaskParameter="Dependencies" ItemName="_BlazorResolvedAssembly" />
+      <Output TaskParameter="Dependencies" ItemName="_BlazorResolvedAssemblyUnlinked" />
     </ResolveBlazorRuntimeDependencies>
 
+    <!--
+    Workaround for https://github.com/dotnet/aspnetcore/issues/19926. Using the files from their inital locations
+    as-is causes RazorSDK to remove files from a hosted app's publish directory. This is because files being removed
+    are looked up by their path. We'll copy files to an intermediate location to trip the removal code.
```

Comment:
```suggestion
    are looked up by their path. We'll copy files to an intermediate location to avoid the removal code.
```

Reply:
Just because I'm not sure what "trip" means in this context (often it would mean what I think is the opposite of the intent here, i.e., trigger the removal code)